
Turn 1
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", line 150, in <module>
    for ship in enemy_ships.extend(my_ships):
TypeError: 'NoneType' object is not iterable

